FMI2QGIS/ui/main_dialog.py

```diff
@@ -102,9 +106,31 @@ def __refresh_stored_wfs_queries(self):
             id_item.setToolTip(sq.id)
             self.tbl_wdgt_stored_queries.setItem(i, 2, id_item)
 
+
+    def __search_stored_wfs_layers(self):
+
+        self.stored_queries: List[StoredQuery] = self.sq_factory.list_queries()
+        self.tbl_wdgt_stored_queries.setRowCount(len(self.stored_queries))
+        self.tbl_wdgt_stored_queries.setColumnCount(3)
+        self.search_string = self.search_ln_ed.value()
+
+        for i, sq in enumerate(self.stored_queries):
+            string_to_match = sq.title.lower()
```

Comment:
Should the search use abstract and id text as well?

Reply:
I originally left them out because the text or the keywords are similar in all of the fields (and for simplicity). But since they are not 100% same, you are right this could be useful. I will look into it.